### ISSUE TITLE
Pin a selected annotation track on top of the window

### ISSUE THREAD

**Issue by aksenia** [igvteam/igv.js#1339]
Hello, 
It would be great if it was possible to pin a selected annotation track (for example, variants in a sample, a vcf/bed) on top/bottom of the annotation view. The selected annotation track would essentially receive its own window, unaffected by scrolling of the rest of annotation, as shown on the screenshot for bigWig window, which is "pinned" with respect to the annotations (A - original view, B - scrolled down).

![issue_pintrack](https://user-images.githubusercontent.com/6505876/120323259-9cdd9a00-c2e5-11eb-8277-4aeac0d35afa.png)

**Comment by jrobinso** [comment]
@aksenia Yes agreed, challenging though as its the browser that is controlling scrolling.   Technically all of our tracks are pinned relative to the document.   Ideas on how to do this are welcome,  probably though configuration rather than end user actions.
